website/docs/r/notification_destination.html.markdown

```diff
@@ -243,39 +243,40 @@ resource "newrelic_notification_destination" "foo" {
 
 In order to create a Slack destination, you have to grant our application access to your workspace. This process is [based on OAuth](https://api.slack.com/authentication/oauth-v2) and can only be done through a browser.
 As a result, you cannot set up a Slack destination purely with Terraform code.
-You can either create a Slack destination in the interface and [import](#import) it to Terraform, or use destination data source.
-
+You can either use [the destination data source](https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/data-sources/notification_destination),
+or reference destination id directly. Destination ID which can be found through [the destination data source](https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/data-sources/notification_destination),
+our GraphQL API, or using the user interface.
 
 ## Import
 
+~> **WARNING:** Please be aware that while you can import the Slack destination resource, it can only be used to delete it through Terraform.
+
```

Comment:
This is nice, but would you want to consider keeping this a little more elaborate, in order to help convey that it isn't really useful to use this destination via Terraform (along the lines of what I've posted [here](https://github.com/newrelic/terraform-provider-newrelic/pull/2661#discussion_r1607150178))? Just give this a thought :)

Reply:
I changed the wording a bit given that the warning is inside the import section so there's no need to talk about configuring or creating a destination. 
Regarding the data source, we already talk about it on the slack section so I removed it as I wanted to make the warning as clear as possible.

Reply:
https://github.com/newrelic/terraform-provider-newrelic/pull/2661#discussion_r1608398693